src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/FixedTermRecallsService.kt

```diff
@@ -31,13 +30,27 @@ class FixedTermRecallsService(private val featureToggles: FeatureToggles) {
 
     val fixedTermRecallSentences = getFixedTermRecallSentences(sentences)
 
-    val sledSentence = getSledSentence(fixedTermRecallSentences, latestExpiryDate)
-    val previousSentence = getPreviousSentence(fixedTermRecallSentences, sledSentence)
+    if (!ftrIsMixedDuration(fixedTermRecallSentences)) {
+      return latestReleaseDate
+    }
+
+    val latestSentence = getLatestSentence(fixedTermRecallSentences) ?: return latestReleaseDate
 
-    return if (sledSentence != null && previousSentence != null) {
-      findMixedDurationReleaseDate(sledSentence, previousSentence, returnToCustodyDate)
+    val maxPostRecallDate = returnToCustodyDate
+      .plusDays(if (latestSentence.recallType == FIXED_TERM_RECALL_14) 13 else 27)
+      .plusDays(latestSentence.sentenceCalculation.adjustments.ualAfterFtr)
+      .plusDays(latestSentence.sentenceCalculation.adjustments.awardedDuringCustody)
+
```

Comment:
the above lgtm, but might be worth @ldlharper  having a quick glance too, I'm just wondering if any other types of adjustments need to be applied